girder/utility/hash_state.py

```diff
@@ -27,7 +27,8 @@
 import binascii
 import sys
 
-_HASHLIB_INLINE_EVP_STRUCT = (sys.version_info < (2, 7, 13) or sys.version_info >= (3,))
+_ver = sys.version_info
+_HASHLIB_INLINE_EVP_STRUCT = (_ver < (2, 7, 13) or _ver >= (3,)) and _ver < (3, 5)
```

Comment:
For consistency with checking the minor version of 2.7, let's check the minor version of 3.5. The changelog shows that the relevant commit is in 3.5.3: https://docs.python.org/3.5/whatsnew/changelog.html#python-3-5-3.